Makefile

```diff
@@ -16,24 +14,29 @@ help:
 	${MAKE} build
 
 build:
-	-rm .docker-build
-	${DOCKERCOMPOSE} -f docker-compose.yml build irc
+	docker build -t standup_irc .
 	touch .docker-build
 
 run: .docker-build
-	${DOCKERCOMPOSE} up irc
+	docker run --env-file .env_dev -v "$PWD:/app" standup_irc
```

Comment:
hmm... there wasn't a postgres service setup in the docker-compose.yml file. I don't really think it's very useful locally, though I could see an argument for having that option. I think the bot does just fine w/o a DB if there isn't one, though testing the DB stuff is important. This is a good question. Perhaps we should keep the compose file even if we don't use it in the Makefile for when one might want to do a full-stack run locally. Or is it possible to switch over to SQLite for local?